modules/social_features/social_core/social_core.install

```diff
@@ -1526,3 +1526,11 @@ function social_core_update_11005(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Add dashboard permission to SM/CM.
+ */
+function social_core_update_11006(): void {
```

Comment:
Shouldn't this permission be added on install instead of update (or maybe both?) ?

Reply:
Good point, added on both ;)